sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateKeyOptions.java

```diff
@@ -26,7 +26,7 @@
     /**
      * Expiry date in UTC.
      */
-    private OffsetDateTime expires;
+    private OffsetDateTime expiresOn;
```

Comment:
Had similar feedback for DateTime in core.
If it is DateTime should it be  `expiresAt`?

Reply:
Nothing came up in arch board, regarding that.
Storage & Identity are also using On suffix.
This change hasn't been standardized in our guidelines. 
So, being consistent with other SDKs for now.